Sources/PackageDescription/PackageDescription.swift

```diff
@@ -324,44 +337,68 @@ public struct LanguageTag: Hashable {
 extension LanguageTag: RawRepresentable {
     public var rawValue: String { tag }
 
+    /// Creates a new instance with the specified raw value.
+    ///
+    /// If there's no value of the type that corresponds with the specified raw
+    /// value, this initializer returns `nil`.
+    ///
+    /// - Parameters:
+    ///   - rawValue: The raw value to use for the new instance.
     public init?(rawValue: String) {
         tag = rawValue
     }
 }
 
+/// ExpressibleByStringLiteral implementation.
```

Comment:
I don't think docc is able to generate documentation for this line. Alternatively, you can use something like

```
// MARK: ExpressibleByStringLiteral Conformance
```

if the intention is to help a reader find this extension quickly.

Reply:
Let me experiment with this a bit. If DocC does not like it, then it's a bug on DocC, so I am inclined to file a bug and leave this as-is.